### PR TITLE
Update security policy

### PR DESCRIPTION
**Pull request by kroeckx**

### CODE REVIEW

**Comment by richsalz** [comment]
FWIW, I like this.  "toward" not "towards" as a language nit, tho.

**Comment by kroeckx** [comment]
What I find online is that they are both correct, and the US seems to prefer toward, while the UK towards.

**Comment by mattcaswell** [comment]
LGTM.

> FWIW, I like this. "toward" not "towards" as a language nit, tho.

"towards" sounds fine to me.

File: policies/secpolicy.html [hunk 1]
```diff
@@ -38,21 +38,44 @@ <h2>Issue triage</h2>
...
 
+	    <h2>Threat Model</h2>
+
+            <p>Certain threats are currently considered outside of the scope of the OpenSSL threat model.
+              Accordingly, we do not consider OpenSSL secure against the following classes of attacks:</p>
+              <ul>
+                <li>same physical system side channel</li>
+                <li>CPU/hardware flaws</li>
+                <li>physical fault injection</li>
+                <li>power consumption observations</li>
```
Comment (tomato42) [inline policies/secpolicy.html]: EM emissions?

and I'm guessing that the sound side-channel falls under power consumption?

**Comment by tomato42** [comment]
> same physical system side channel

I still think that this need to be separated from the other ones and it's a reasonable requirement for deployment on modern systems (IaaS)

yes, the code may not be secure against it, but those are vulnerabilities that people care about thus they should be given CVE numbers and fixed as a priority

**Comment by kroeckx** [comment]
Can you explain a little more what you mean with "separated from the other ones"?
The text saying that we will fix it is not enough?

Would it help if I reword it to say the attacker needs to be able
the execute code on the same physical system, versus those where
the attacker either has access to the hardware or can monitor
things like power consumption?

**Comment by tomato42** [comment]
yes, that's what I meant – code execution by attacker can happen remotely, all the other attacks require high degree of physical access, proximity or control over the machine under attack

**Comment by t8m** [comment]
The problem is that even physical fault injection or CPU flaws can be used in situations where physical access is not required. And on the other hand aggressive sharing of CPU resources through hyperthreading can be regarded as kind-of CPU or operating environment flaw. So the separation is not that clear.

File: policies/secpolicy.html [hunk 2]
```diff
@@ -38,21 +38,44 @@ <h2>Issue triage</h2>
...
 
 	    <ul>
               <li><em><a name="critical">CRITICAL</a> Severity.</em>
               This affects common configurations and which are also likely to
               be exploitable. Examples include significant disclosure of the
               contents of server memory (potentially revealing user details),
               vulnerabilities which can be easily exploited remotely to
-              compromise server private keys (excluding local, theoretical or
-              difficult to exploit side channel attacks) or where remote code
+              compromise server private keys
```
Comment (paulidale) [inline policies/secpolicy.html:77]: Join the two short lines?

File: policies/secpolicy.html [hunk 3]
```diff
@@ -67,20 +90,20 @@ <h2>Issue severity</h2>
...
 	      can roll up several such flaws at one time.</li>
-              
+
 	      <li>
 	      <em><a name="low">LOW</a> Severity.</em>
 	      This includes issues such as those that only affect the
-	      openssl command line utility, unlikely configurations, or hard
-	      to exploit timing (side channel) attacks.  These will in general
+	      openssl command line utility, or unlikely configurations.
+	      These will in general
```
Comment (paulidale) [inline policies/secpolicy.html:106]: Avoid the short line.

File: policies/secpolicy.html (hunk 2 above)
Comment (paulidale) [reply policies/secpolicy.html:77]: HTML -- it doesn't matter

File: policies/secpolicy.html (hunk 1 above)
Comment (paulidale) [reply policies/secpolicy.html]: Would changing the final item to _physical observation side channels_ be better?
Or _physical observation side channels (e.g. power consumption, EM emissions, etc)_ be better?

Comment (tomato42) [reply policies/secpolicy.html]: yes, I like the latter more (though I'm not sure if combining "e.g." with "etc." is stylistically acceptable)

Comment (paulidale) [reply policies/secpolicy.html]: @kroeckx is this change likely to be included?

File: policies/secpolicy.html [hunk 4]
```diff
@@ -12,7 +12,7 @@
 	  <header>
 	    <h2>Security Policy</h2>
 	    <h5>
-	      Last modified 16th May 2018
+	      Last modified 29th November 2018
```
Comment (t-j-h) [inline policies/secpolicy.html:15]: When performing the merge please update this to the closing date of the policy change vote.

**Review by paulidale — APPROVED** [review]

**Review by mattcaswell — APPROVED** [review]

**Review by levitte — APPROVED** [review]

**Review by t-j-h — APPROVED** [review]

**Review by vdukhovni — APPROVED** [review]

**Review by iamamoose — APPROVED** [review]

**Pull request closed without merging**